libs/sdk-ui-filters/src/DateFilter/utils/Translations/DateFilterTitle.ts

```diff
@@ -228,8 +221,12 @@ const getDateFilterRepresentationUsingTranslator = (
     }
 };
 
-export const getDateFilterRepresentation = (filter: DateFilterOption, locale: ILocale): string => {
+export const getDateFilterRepresentation = (
+    filter: DateFilterOption,
+    locale: ILocale,
+    dateFormat: string,
+): string => {
     const translator = getIntl(locale);
 
-    return getDateFilterRepresentationUsingTranslator(filter, translator);
+    return getDateFilterRepresentationUsingTranslator(filter, translator, dateFormat);
```

Comment:
it seem that `dateFormat` can be replace for `translator`. But here we use both `translator` and `dateFormat`.
Should we use only one of them?

Reply:
dateFormat is used for absolute date
translator is used for relative date (i18l)
So we can't use only 1